### DISCUSSION TITLE
GitHub Certifications Feedback

### DISCUSSION THREAD

**Discussion by omenking** [community/community#87415] [Other Feature Feedback, Questions, & Ideas]
Hey GitHub, 

I'm providing feedback for the GitHub Foundations and general feedback about the GitHub Certifications.

I think the certifications are a great idea, and I see alot of value for folks learning GitHub.

I did spot some issues which I hope can get resolved or choices I think I'd like to convince you to change.

### Lack of Course Code

Please get official course codes that indicate the version of the exam.

Most Certification will have course codes and as a content creator I can tell you students use the course codes to know if an exam is the most current version and will use it as a search term to find study materials.

I was able to implicitly determine course codes by the initialisms being used in the links for the certifications:
- GitHub Foundations (GHF)
- GitHub Adminstrator (GHA)
- GitHub Advanced Security (GHAS)
- GitHub Actions (???)

Ideally I'd like to see something like GHF-100 or something that indicates version.

### Lack of Domain Breakdowns

Domains are the sections of the exam.

- Domain 1: Support GitHub Enterprise for users and key stakeholders
- Domain 2: Manage user identities and GitHub authentication
- Domain 3: Describe how GitHub is deployed, distributed, and licensed
- Domain 4: Manage access and permissions based on membership
- Domain 5: Enable secure software development and ensure compliance
- Domain 6: Manage GitHub Actions
- Domain 7: Benefits of the GitHub Community

Certification exams will tell you the percentage of questions that will appear on your exam.
For someone studying for exams the domain percentages helps avoid under or over study specific domains.

GitHub Certification provides no such percentages.

My experience for my exam sitting for the pool of questions based on given domains was unbalanced.

I had a significant number of questions about GitHub Enterprise Server and GitHub Enterprise Cloud.

Based on the exam guide outline, I did not expect this many questions.
I did not expect the depth of the questions to be asked.

Since there is a GitHub Admistration certification, I would have thought more in-depth questions would be left to that exam.

### Lack of Passing Score

Certification exams will tell you how many points are in the exam and how many you need eg. 700 / 1000 which would roughly be 70%. Since exams used Scaled Scoring, it's understood that you could technically get 70% and still fail due to the distribution of questions and possible points for questions.

GitHub has decided not to share the passing score.

## Missing Content

When I operated my bootcamps I noticed there was a significant gap with:
- developer skills
- git skills
- git and project workflow skills.

Technical professionals without developer backgrounds showed a strong interest in picking up these skills to round their skill set to the point that a Git Bootcamp was highly requested.

### Git Basics Section Missing

If you want to use GitHub you need some Git skills.
I would think that there would be a bare-bones primer front-loaded into this GitHub Foundations, and the exam would validate this knowledge.

The only exam question I validated git knowledge was `git checkout -b branch-name`

### Developer Section Missing

Developer tools are completely absent in all the GitHub Certification exam outlines.

Things I would have expected in GitHub Foundations:
- GitHub API (REST and GraphQL)
- GitHub SDKs (Octokit, Ruby, JS, Terraform)
- GitHub CLI
- PATs 
- SSH Keys and Deploy Keys
- GitHub Flavoured Markdown
- Tagging
- devcontainer.json configuration

> There is some implicit use of developers' tools such as PATs, but they oddly get no explicit attention.

And yet, we have exam questions asking us to know the structure of GitHub Actions workflow files.

I would have thought the concepts of GitHub Actions would have been enough in GitHub Foundations and technical-specific questions would be left for the GitHub Actions Certification.

## Format of Questions

The questions are multiple-choice and multiple-select, which is common for foundation/fundamental certifications.

The length of questions and choices were of expected length.

What I think will frustrate or annoy test takers is the fact that this is a very broad exam, and yet the exam decides to go too deep, asking about technical specifics of GitHub features.

Example: In Github Project Insights, a question would want you to know the types of charts provided (Historical and Current charts). Why isn't it enough to know it has graphs? Why get that detailed on this specific feature?

Example: When using Github Issues, it will ask you to choose the correct advanced search syntax out of a lineup with multiple filters. Why isn't it enough to know how robust the search is? Why do I have to lose a point because I can't recall the exact advanced search syntax filter syntax?

Then there are questions where there are technically more correct answers. This can happen in Professional certifications, but normally, Foundations/Fundamentals certification format their choices as Horse, Horse, Zebra, Elephant.

I could write about 30 examples I can recall, But I don't want to be creating an exam dump, so just take my word that the questions need work.

## Beta Features in Outline and Exam

I am confused about the inclusion of Beta features in exams.

Generally, certification will only have GA features, yet GitHub includes them.

~~One in particular that I could not get working was GitHub Issue Forms.~~

**Comment by NoSQLKnowHow** [comment]
Having been a consumer of @omenking's boot camp as well as a guest "teacher" on a module for one of his boot camps, he definitely knows what he is talking about. This post is bang on as constructive criticism for the recently released GitHub certs!

**Reply by Cynthia7374** [reply]
How

**Reply by Cynthia7374** [reply]
> How

Humm

**Comment by ghostinhershell** [comment]
Thanks so much for sharing your feedback here and welcome to our Community @omenking! 🎉 

I've shared this feedback with the applicable teams and will follow-up here if we have any additional questions or updates on our end. Thanks again!

**Comment by Paul-Stetsenko** [comment]
Great points @omenking 😎  and thanks for your course!

**Comment by BrendaThistle** [comment]
@omenking thank you so much for your feedback (we did also receive your email), we are going through it and will catch up with you next week on our call.

**Comment by matiasrodlo** [comment]
@omenking thanks for your course!

**Comment by gsbad** [comment]
Great course! Thx bro!

**Comment by vickneee** [comment]
Thank you for the course! Very useful!

**Comment by Jakhaishya** [comment]
thank you for the course

**Reply by PrakharChauhan1739** [reply]
great

**Comment by averyaaa1** [comment]
Could not have said these critiques better! GitHub listen to him!

**Comment by Adongo** [comment]
Very useful course

**Comment by or-authentrip** [comment]
I think the fellows who work in GitHub certifications should review your [course recordings](https://youtu.be/mQsjPatWwpo?si=U11--FoqNkClH_Dh) and make notes based on your feedback. Thanks for the awesome course, Andrew!

**Comment by itzzjb** [comment]
Thanks for the course @omenking.  I just passed the exam in 2 days just by your video.

**Comment by Steven-Herrera** [comment]
I have seen several videos created by @omenking, and I would advise GitHub integrate his suggestions into their certifications. He definitely knows what he is talking about!

**Comment by 139593-77** [comment]
Hi Github,
I have attempted Github Foundation certification  and attempted all 75 questions  but in the result it is showing out of 60 instead of 75, can you please explain this ?

**Reply by BrendaThistle** [reply]
Hi there, can you forward your result email to gh-certification@github.com and we can look into it?

**Reply by sushantab0318** [reply]
There are 60 scored questions and 15 unscored or pretest questions. They don't tell which one is scored and which one is not.

**Reply by PRIYABRATA2** [reply]
I have a doubt, when appearing can we give answers to only 60 questions out of 75 , as its getting evaluated out of 60. If we appear all questions , may be initial 60 questions will get considered. Any thoughts ?

**Reply by BrendaThistle** [reply]
@PRIYABRATA2 you'd have to answer all 75.

**Comment by Sahilsr10** [comment]
What will happen if someone is unable to show up on the day of exam?

**Reply by BrendaThistle** [reply]
Per the [FAQs](https://examregistration.github.com/faq): If my plans change, and I cannot sit for the exam, what should I do?
You can cancel or reschedule your exam for any reason up to 48 hours before your scheduled appointment. If an unforeseen circumstance happens within that window, we recommend reaching out to GitHub Support for assistance. To do so, please open a ticket using this [link](https://support.github.com/contact?tags=hh-ace-support). We kindly request that you do not select the option to chat with Copilot in GitHub Support.

**Comment by 139593-77** [comment]
TCS Confidential

Can you please give me proper email I'd? Link you have given is http link and not email address

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by BrendaThistle** [reply]
Apologies! It's gh-certification@github.com

**Comment by 139593-77** [comment]
TCS Confidential

Have sent it

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 139593-77** [comment]
TCS Confidential

After clicking on End Test button, subsequent page shows me result as 35/60 even though I attempted all 75 questions

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by 139593-77** [comment]
TCS Confidential

ANY update on below email pls ?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by omkarjoshi1989** [comment]
Hello Github support,
After attempting all 75 question answers, and ending the exam it shows 35/60 result. Why its was shown 60 and not 75 total?

**Reply by AlexHedley** [reply]
From their docs
> Format: Our current exams are: multiple-choice, have 60 scored items, and around 10-15 pretest items depending on the exam.

**Comment by ssh-juan** [comment]
@omenking Thanks for your contents!

**Comment by KunjeshRamani** [comment]
Good Efforts. Thank you @omenking

**Comment by Chirantan-97** [comment]
Dear PSI Support Team,

I am writing to express my extreme dissatisfaction and frustration regarding the issues I faced during my GitHub Foundation certification exam on 27th July, 2024 at 18:00 hrs IST. My exam confirmation code is  C124090504

I adhered to all instructions and successfully completed the check-in process. Throughout the exam, I answered all the questions without any issues. However, when attempting to submit my answers, I encountered an error stating "no internet connection." This is perplexing as I was using a reliable broadband connection with excellent speed, and the camera streaming was functioning perfectly.

Despite following the technical assistant team's advice to relaunch the exam three times, the issue persisted. Ultimately, I was instructed by a technical assistant to end the session, which I did. Subsequently, I received an email stating that I had failed the exam and scored zero. This outcome is entirely unjust and unacceptable.

My ticket numbers are as follows : 4343823 ; 4343876

I am extremely disappointed by the lack of effective technical support. I repeatedly requested assistance, yet no resolution was provided. This situation has resulted in the loss of my valuable time, my exam voucher, and has caused significant mental distress due to the effort and dedication I invested in preparing for this exam.

I demand an immediate resolution to this matter. I request that you either issue me a new exam voucher or process my actual exam results based on the answers I submitted. Failure to address this promptly will compel me to take strict action against PSI for the negligence and the undue stress caused.

I expect a prompt response and resolution to this matter.

Sincerely,

CHIRANTAN GHOSH
** Post edited by moderator to remove PII **

**Comment by Chirantan-97** [comment]
****Dear PSI Support Team,

I am writing to express my extreme dissatisfaction and frustration regarding the issues I faced during my GitHub Foundation certification exam on 27th July, 2024 at 18:00 hrs IST. My exam confirmation code is C124090504

I adhered to all instructions and successfully completed the check-in process. Throughout the exam, I answered all the questions without any issues. However, when attempting to submit my answers, I encountered an error stating "no internet connection." This is perplexing as I was using a reliable broadband connection with excellent speed, and the camera streaming was functioning perfectly.

Despite following the technical assistant team's advice to relaunch the exam three times, the issue persisted. Ultimately, I was instructed by a technical assistant to end the session, which I did. Subsequently, I received an email stating that I had failed the exam and scored zero. This outcome is entirely unjust and unacceptable.

The lack of effective technical support is appalling. I repeatedly requested assistance, yet no resolution was provided. This has resulted in the loss of my valuable time, my exam voucher, and has caused significant mental distress due to the effort and dedication I invested in preparing for this exam.

I demand an immediate resolution to this matter. I insist that you either issue me a new exam voucher or process my actual exam results based on the answers I submitted. Failure to address this promptly will compel me to escalate this matter and take strict legal action against PSI for gross negligence and the undue stress caused.

This situation is utterly unacceptable, and I expect an urgent response. Any further delay or failure to resolve this issue satisfactorily will leave me with no option but to seek redress through all available legal and regulatory channels. I expect this matter to be resolved within 48 hours.

Sincerely,

CHIRANTAN GHOSH
** Post edited by moderator to remove PII **

**Comment by mattpickles1** [comment]
Having completed (almost) all of the GitHub introductory training before the certfications existed and having reviewed the GitHub foundations training path that exists now, I couldn't honestly say I would have passed the certification but the course provided by @omenking certainly provided me with enough knowledge to pass the GitHub foundations exam (and I was grateful for the reduced exam fee offered by GitHub).

In the case of the GitHub Actions certfication, I'm still struggling despite notionally working in an environment where these functionalities/tools are used, having similar certifications from other providers, having done both the GitHub training path and the course provided by @omenking .

Of course, this is my "problem" and it's certainly not the first time I've failed a certifcation exam but I've never been left so bewildered after an exam about what I actually need to know or where I need to reinforce my knowledge. 

I agree with points made by @omenking  to greater or lesser degree and what I certainly do find is that the GitHub certifications are at least in this initial offering not as clear as certifications from other providers regarding exam content and what knowledge you really need to have to pass.

**Comment by AdamIbrahim85** [comment]
Hi Mate, 

I appreciate the work you have put in to help a noob like myself get his head around things. I still have a few hours worth of video to go through and I usually squeeze in time here and there.

My issue is I tried booking the certification through psi. What a disaster that has been. they took my money and no time slot booked. Now I am waiting for customer service but the last 3 replies from them all keep saying to call numbers which are disconnected..... 

If anyone has any advice on how to proceed please let me know. They are so unprofessional I would class them as a scam but they are linked through github so you would think they are legit.

Are there any other providers you can take the exam through other than psi?

**Reply by BrendaThistle** [reply]
Hi @AdamIbrahim85 please reach out to gh-certification@github.com and they can assist you!!

**Comment by TIAGOLINHARES** [comment]
@omenking thanks for your course!

**Comment by sushantab0318** [comment]
Good job! Really appreciate your effort.

**Comment by sushantab0318** [comment]
That would be nice  but I believe there are  specific questions which is non evaluated. They pop up randomly. Let’s say you attend only first 60 and got 7 of those useless questions and 3 wrong answers then you score 50 out of 60. On Jan 23, 2025, at 4:52 PM, PRIYABRATA2 ***@***.***> wrote:﻿
I have a doubt, when appearing can we give answers to only 60 questions out of 75 , as its getting evaluated out of 60. If we appear all questions , may be initial 60 questions will get considered. Any thoughts ?

**Comment by PRIYABRATA2** [comment]
any idea, then only first 60 questions are getting evaluated and next 15
not ?

On Fri, Jan 24, 2025 at 3:09 PM Brenda Thistle ***@***.***>
wrote:

> @PRIYABRATA2 <https://github.com/PRIYABRATA2> you'd have to answer all 75.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/87415#discussioncomment-11944397>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APJOMVXBHOD7CQLMSCRWFDT2MJJS5AVCNFSM6AAAAABB2TW4JGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOJUGQZTSNY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Priyabrata Samantaray
Mob No-9439902551

**Reply by BrendaThistle** [reply]
The non-scored items are randomly placed throughout the exam.

**Comment by PRIYABRATA2** [comment]
Thanks for the clarity.

**Comment by gcpKushan** [comment]
Very useful, Thanks Andrew.

**Comment by atulgosain** [comment]
I am really enjoying the learning, thanks for preparing it for us. Cheers!!

**Comment by jyothimaniR** [comment]
@omenking may not have all the answers, but he’s got great vibes and Google ✨😂

**Comment by shrey-2-gupta** [comment]
Great work in the course!

**Comment by aiford1** [comment]
**PassExam4Sure** is reliable and exam-focused. It helped me understand both theory and exam format. Passed successfully.

**Comment by Kritzar** [comment]
Hi can anyone help me i gave my Github Copilot certification exam on 18 April and i passed that test but did not recieved any certificate can anyone tell me if they had same experience did they recieved certificate a day later or so, or where should i contact regarding this ?

**Reply by Dmistry1901** [reply]
Hello Kritzar, I had given GH-900 as well and had passed the exams. But I too was not given Credly Badge and Score Card.
You can try and mail **`gh-certification@github.com`** but they have stopped sharing the credly badge and **certificate can be downloaded from Microsoft Learn**.

**Reply by Kritzar** [reply]
I am unable to see my certification in my Learn profile.

**Reply by lanape** [reply]
Hi Kritzar and Dmistry1901. You’re right. We no longer provide Credly badges for GitHub certifications. Instead, you’ll be able to download your certificate directly from your Microsoft Learn profile.

At the moment, you won’t see this certification reflected in GitHub Learn yet, as we’re still working on the integration between both platforms.

If you have additional questions, don't hesitate to reach out our GitHub Certification Team gh-certification@github.com

**Comment by qa4examofficial** [comment]
### Top GitHub Exam Questions & Answers

QA4Exam provides the **most accurate & verified GitHub Exam Questions** & Answers in PDF and Online Practice Test formats, Our GitHub Exam Questions & Answers are regularly updated to meet the accuracy of Exam Questions & Answers in any of your actual GitHub Exam. We guarantee the quality and reliability of our resources, assuring that they are the ideal tool for passing the GitHub Exam.

**https://www.qa4exam.com/github-qa**
